src/user_interface.py

```diff
@@ -609,20 +618,28 @@ def __confirm(self):
             self.__ready_to_continue.set(1)
 
     def __show_help(self):
-        helpfile = str(Path(__file__).parent / "assets" / "manual.pdf")
-        subprocess.Popen([helpfile], shell=True)
+        helpfile = helpfile = os.path.join(Path(__file__).parent, "assets",
+                "manual.pdf")
```

Comment:
Are you sure `os.path.join` is necessary here? Notice that in the original code we are not combining strings but instead using the slash (`/`) operator, which should join paths similarly to `os.path.join`. I think that the `/` way is more readable if it does work on all systems.

For reference, [here's the documentation](https://docs.python.org/3/library/pathlib.html#operators) on the slash operator.

Reply:
I have not investigated the newer `path` module. However, I am guessing it rides on top of `os.path`. In interpreted code I generally try to avoid abstractions as much as possible. I know that `os.path.join` works across all platforms. It does look like your usage of `path` works across all platforms as you use it other places as well, but I am not confident of how robust it is. I also am wary of overloading fundamental operators such as divide `/`.

Reply:
I think I would prefer to keep this style as it's shorter and more readable, and to be consistent across the whole app. The `path` module is the more modern standard solution and should work in all situations.